talisker/sentry.py

```diff
@@ -58,6 +59,11 @@ def register_client_update(update_func):
     return update_func
 
 
+def update_clients(client):
```

Comment:
Well, you are updating all the references to the client, so I guess update_client_references?